packages/sdk/wallets/wallet-base/src/signing-utils.ts

```diff
@@ -91,14 +92,14 @@ function makeEven(hex: string) {
 
 function signatureFormatter(
   signature: { v: number | bigint; r: Buffer; s: Buffer },
-  type: TransactionTypes
+  type: TransactionTypes | 'celo-legacy'
```

Comment:
i wanted to avoid adding it back as a first class supported type so did this instead